coq-builtin.elpi

```diff
@@ -1144,6 +1144,9 @@ external pred coq.TC.db-tc o:list gref.
 % [coq.TC.db-for GR Db] reads all instances of the given class GR
 external pred coq.TC.db-for i:gref, o:list tc-instance.
 
+% [coq.TC.db-for2 GR List Db] reads all instances of the given class GR
+external pred coq.TC.db-for2 i:gref, o:list gref.
```

Comment:
I was talking about this API.
If you can return tc-instance instead of gref, we can just replace the old API above.

Reply:
Yep, I can do so